mlops-multi-account-cdk/mlops-sm-project-template/seed_code/deploy_app/upload_assets.py

```diff
@@ -0,0 +1,69 @@
+'''
+loop through CDK generated assets and upload to s3
+
+'''
+from os import path, listdir, getenv
+import boto3, shutil
+
+import logging
+logging.basicConfig()
+
+logger = logging.getLogger(__file__.split('/')[-1])
+logger.setLevel(getenv("LOGLEVEL", "INFO"))
+
+def upload_assets_to_s3(account_id):
```

Comment:
Hey Maria, do we actually need that part?
Can we just keep the pipeline definition and code in the dev account and just point to that file in "deploy_endpoint_stack"?
The fact of copying the pipeline definitions and scripts could be avoided as long as the Role for preprod and prod has access to the s3 bucket in the dev account